tools/walletextension/multi_acc_helper.go

```diff
@@ -175,11 +176,27 @@ func proxyRequest(rpcReq *rpcRequest, rpcResp *interface{}, we *WalletExtension)
 }
 
 func executeCall(client *rpc.EncRPCClient, req *rpcRequest, resp *interface{}) error {
-	var err error
+	if req.method == rpc.RPCSubscribe {
+		if len(req.params) == 0 {
```

Comment:
This branch is completely different from the rest of the executeCall code, might be better to `return executeSubscribe(...)`  and move that block in there. And maybe even consider if that branch of Call vs Subscribe could have happened earlier. (Could maybe add an `execute()` method that just branches between `Call` and `Subscribe` and gets used from `proxyRequest`)